presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -666,12 +668,33 @@ output expressions:
 
     ORDER BY expression [ ASC | DESC ] [ NULLS { FIRST | LAST } ] [, ...]
 
-Each expression may be composed of output columns or it may be an ordinal
-number selecting an output column by position (starting at one). The
-``ORDER BY`` clause is evaluated after any ``GROUP BY`` or ``HAVING`` clause
+Each expression may be composed of output columns, or it may be an ordinal
+number selecting an output column by position, starting at one. The
+``ORDER BY`` clause is evaluated after any ``GROUP BY`` or ``HAVING`` clause,
```

Comment:
i don't get the "when to put comma" rules, but i trust you on this.

Reply:
whenever a part is a full sentence, and you continue on anyway with basically a new sentence, you use a comma. That is one of the rules.